protected/controllers/PatientController.php

```diff
@@ -1507,8 +1507,8 @@ public function actionCreate()
             $patient->attributes = $_POST['Patient'];
             $address->attributes = $_POST['Address'];
 
+            $referral = new PatientReferral();
```

Comment:
Does this have any negative effect on editing an existing patient with a referral?